src/main/kotlin/no/nav/familie/ef/sak/service/TilkjentYtelseService.kt

```diff
@@ -11,32 +11,37 @@ import no.nav.familie.ef.sak.api.dto.TilkjentYtelseDTO
 import no.nav.familie.ef.sak.repository.TilkjentYtelseRepository
 import no.nav.familie.ef.sak.integration.FAGSYSTEM
 import no.nav.familie.ef.sak.integration.ØkonomiKlient
+import no.nav.familie.ef.sak.repository.BehandlingRepository
 import no.nav.familie.ef.sak.økonomi.UtbetalingsoppdragGenerator.lagTilkjentYtelseMedUtbetalingsoppdrag
 import no.nav.familie.kontrakter.felles.getDataOrThrow
 import no.nav.familie.kontrakter.felles.oppdrag.OppdragId
 import no.nav.familie.kontrakter.felles.oppdrag.OppdragStatus
 import org.springframework.data.repository.findByIdOrNull
 import org.springframework.stereotype.Service
 import org.springframework.transaction.annotation.Transactional
+import java.lang.IllegalStateException
 import java.time.LocalDate
 import java.util.*
 
 @Service
 class TilkjentYtelseService(private val økonomiKlient: ØkonomiKlient,
-                            private val tilkjentYtelseRepository: TilkjentYtelseRepository) {
+                            private val tilkjentYtelseRepository: TilkjentYtelseRepository,
+                            private val behandlingRepository: BehandlingRepository) {
 
     @Transactional
     fun opprettTilkjentYtelse(tilkjentYtelseDTO: TilkjentYtelseDTO): UUID {
         tilkjentYtelseDTO.valider()
-        val saksbehandlerId = SikkerhetContext.hentSaksbehandler()
 
         val eksisterendeTilkjentYtelse = tilkjentYtelseRepository.findByPersonident(tilkjentYtelseDTO.søker)
         if (eksisterendeTilkjentYtelse != null) {
             error("Søker har allerede en tilkjent ytelse")
         }
 
+        val behandling = behandlingRepository.findById(tilkjentYtelseDTO.behandlingId)
```

Comment:
Jeg tror det finnes en servicemetode for dette i `BehandlingService` som bruker `findByIdOrThrow`. Så slipper du å bruke repository-metoden direkte her :)